Tests/Fixtures/Generated/XCAssets/swift4/all-allValues.swift

```diff
@@ -239,7 +239,8 @@ internal struct ImageAsset {
     #if os(iOS) || os(tvOS)
     let image = Image(named: name, in: bundle, compatibleWith: nil)
     #elseif os(macOS)
-    let image = bundle.image(forResource: NSImage.Name(name))
+    let name = NSImage.Name(self.name)
+    let image = (bundle == .main) ? NSImage(named: name) : bundle.image(forResource: name)
```

Comment:
For refs: I was wondering if `Bundle` was conforming to `Equatable` in a more specific way that the default `NSObject.==` it is inheriting from (which is just pointer comparison) – which could lead to issues if we were trying to compare `Bundle(path: pathToMain)!` vs `Bundle.main`.

But after some tests it seems that the `Bundle.init` magically caches and shares bundle instances pointing to the same path, so we're good using `==` here anyway, even if it might just use pointer comparison 👍